content/en/getting_started/serverless/_index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Getting Started with Serverless Monitoring
+kind: documentation
+
+further_reading:
+    - link: '/agent/basic_agent_usage/'
+      tag: 'Documentation'
+      text: 'Basic Agent Usage'
+---
+
+## Overview
+
+_Serverless_ is a model where developers build and run applications and services using a cloud provider, rather than managing infrastructure themselves. Datadog [Serverless Monitoring][10] collects metrics, logs, and traces from your serverless infrastructure, enabling you to monitor your application's health and performance.
+
+{{< img src="getting_started/serverless/serverless_view.png" alt="Serverless Monitoring: Serverless View, an explorer page" style="width:80%;">}}
+
+This guide makes use of a serverless [sample app][1] that you can launch with one click. This app has Serverless Monitoring preconfigured. Follow this guide to see how you might troubleshoot a problem in your sample app, and what kinds of visibility Serverless Monitoring can provide.
+
+### Install the sample app
+
+1. [Launch the CloudFormation Stack][9]. This link brings you to a **Create stack** page in CloudFormation.
+2. Enter your Datadog API key and Datadog site ({{< region-param key="dd_site" code="true" >}}). 
```

Comment:
I guess it doesn't hurt to link "Datadog API key" to the right page for convenience? Also for "Datadog site", lots of people don't change the site selector on the doc site when reading public docs, I think it would be nice (at least doesn't hurt) to also link it to the https://docs.datadoghq.com/getting_started/site.